install/linux/build-rpm.py

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 import argparse
+import getpass
 import inspect
 import os
 import pathlib
```

Comment:
nitpick: As we're only using `pathlib.Path` in this file, we might just import it:

```python
from pathlib import Path
```

Reply:
You're right, fixed in bdf0204dd33730fc294c82bcd934a35f940da6e2.